GNUmakefile

```diff
@@ -0,0 +1,2 @@
+include Makefile
+PUBLISHBRANCH = gh-pages
```

Comment:
Please delete this file.  It is intended for your working copy only and should not be committed.  

Its purpose is to setup preview page under your account rather than to diffpy.org.